solarforecastarbiter/io/reference_observations/default_forecasts.py

```diff
@@ -176,3 +176,45 @@ def is_in_nwp_domain(site):
     TEMPLATE_NWP_FORECASTS +
     TEMPLATE_DETERMINISTIC_PERSISTENCE_FORECASTS
 )
+
+TEMPLATE_NETLOAD_PERSISTENCE_FORECASTS = [
+    Forecast(
+        name='Week Ahead Persistence',
```

Comment:
Just a question as I look at the persistence code for something else, right now, net load variables automatically use data from a week before (https://github.com/SolarArbiter/solarforecastarbiter-core/blob/150342b18695710f55a72e5d92da04f890604c4a/solarforecastarbiter/reference_forecasts/utils.py#L233). This means this "weak ahead" forecast is really using data from a week ago for next week, and using data from a week ago for tomorrow. Is that the intention?

I would think the intention here was that the week ahead persistence would capture the daily changes by using data from yesterday for next week (so maybe the lead time should be 6 days) and the day head persistence wouldn't capture the day of week variability but would use data from the previous day.

Reply:
Thanks for catching this. You are correct that this is a mistake and I'll revise to 1) remove the "Week Ahead Persistence" template forecast and 2) rename the "Day Ahead Persistence" as "Day of Week Persistence" (which is more accurate and likely clearer to anyone who sees it in the dashboard).